src/Zicht/Itertools/filters.php

```diff
@@ -193,6 +201,14 @@ function before($expected, $strategy = null, $orEqual = false)
     if ($expected instanceof \DateTimeInterface) {
         return function ($value, $key = null) use ($expected, $strategy, $orEqual) {
             $value = $strategy($value, $key);
+            // Try to convert strings that are longer than 3 characters.  This simple test will ensure
+            // that '' or 'now' will not be accepted.
```

Comment:
Waarom mag 'now' niet geaccepteerd worden?

Reply:
Het gaat om de volgende dataset use-case:
[{title: 'Foo', date: '2020-01-01'}, {title: 'Bar', data: '2020-01-02'}, {title: 'Moo', data: 'now'}]

Als ik een dataset heb, i.e. uit SRO, en ik sla die data op in bijvoorbeeld de een redis cache, dan wil je toch zeker niet 'now' een uur later evaluaren als 'now'?  Eigenlijk wil je alleen strings accepteren die een ISO format hebben.

Misschien is dat nog wel een veel betere check: `preg_match('/^[0-9]{4}-[0-9]{2}-[0-9]{2}', $value)`

Wat denk jij?

Reply:
Ok